gapic-generator/lib/gapic/presenters/gem_presenter.rb

```diff
@@ -134,7 +134,7 @@ def gemspec_description
       def readme_description
         has_markdown = description.strip.start_with? "#"
         desc = has_markdown ? description.split("\n") : [description.gsub(/\s+/, " ").strip]
-        Gapic::FormattingUtils.format_doc_lines @api, desc
+        Gapic::FormattingUtils.format_doc_lines @api, desc, disable_xrefs: true
```

Comment:
Oh. I think we want this in the wrapper readme but the normal gapic readme is okay as is. So I suspect we want this change in an override in gapic-generator-cloud/lib/gapic/presenters/wrapper_gem_presenter.rb.